include/odp/api/spec/traffic_mngr.h

```diff
@@ -385,6 +410,29 @@ typedef struct {
 	 * the parameters of the threshold profile of any TM node or TM queue.
 	 */
 	odp_bool_t dynamic_threshold_update;
+
+	/** Indicates the packet priority modes supported by a TM systems on a
+	 * platform. A platform can support multiple packet priority modes. The
+	 * actual mode, a TM system runs with is defined by
+	 * odp_tm_requirements_t.
+	 */
+	odp_bool_t pkt_prio_modes[ODP_TM_PKT_PRIO_MODE_MAX];
+
+	/** max_schedulers_per_node indicates maximum number of schedulers
+	 * supported by a tm node at any level.
+	 *
+	 * At any node, a scheduler is assigned to each priority to schedule
+	 * all fan-in packets with equal priority from different fan-in nodes.
+	 * This capability indicates that at a given node, max schedulers
+	 * available. This implies at a given node there cannot be more
+	 * than max_schedulers_per_node priorities that have packets from
+	 * multiple fan-in nodes and hence need scheduling.
+	 * Rest of the fan-in nodes should have unique priority packets.
+	 *
+	 * This value can range from 0..ODP_TM_MAX_PRIORITIES as there can be
+	 * only as one scheduler is assigned per static priority.
+	 */
+	uint8_t max_schedulers_per_node;
```

Comment:
The description here is not very clear and has grammar problems. I propose the following text instead:

```
Maximum number of schedulers supported by a TM node at any level.

A TM node contains a WFQ/WRR scheduler for each packet priority level
for which the node has more than one possible input. TM topology and
priority configuration must be made so that the resulting number of
WFQ/WRR schedulers does not exceed this capability in any TM node.

The value can vary between 0 and ODP_TM_MAX_PRIORITIES.
```